content/en-us/cloud-services/memory-stores/best-practices.md

```diff
@@ -151,7 +151,7 @@ end
 
 -- Write some code!
 
-for i,player in pairs(game.Players:GetChildren()) do
+for i,player in game:GetService("Players"):GetPlayers() do
```

Comment:
Nice catch! Any other place you see `pairs()` being used when it's no longer needed, please feel free to submit a PR for it.